src/Compute/Compute/Extension/RemoveAzureVMExtensionCommand.cs

```diff
@@ -58,6 +58,9 @@ public class RemoveAzureVMExtensionCommand : VirtualMachineExtensionBaseCmdlet
         [ValidateNotNullOrEmpty]
         public SwitchParameter Force { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
```

Comment:
put each assignment on their own line like the other parameters.

Reply:
Got it